src/megatron/bridge/utils/common_utils.py

```diff
@@ -39,6 +39,20 @@
     ALL_MODULE_WRAPPER_CLASSNAMES = (DDP, Float16Module)
 
 
+SAFE_REPOS = [
+    "nvidia",
+    "Qwen",
+    "deepseek-ai",
+    "meta-llama",
+    "google",
+    "openai",
+    "mistralai",
+    "moonshotai",
+    "llava-hf",
+    "gpt2",
+]
```

Comment:
common_utils is too broad for this. i'd recommend creating a utils file here: https://github.com/NVIDIA-NeMo/Megatron-Bridge/tree/main/src/megatron/bridge/models/hf_pretrained